src/stories/containers/TransparencyReport/components/HeaderWithIcon/HeaderWithIcon.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+import { CustomPopover } from '@ses/components/CustomPopover/CustomPopover';
+import Information from '@ses/components/svg/information';
+import lightTheme from '@ses/styles/theme/light';
+import React, { useRef } from 'react';
+import HeaderToolTip from './TooltipHeader';
+
+interface Props {
+  description: string;
+  mipNumber: string;
+  link: string;
+  name: string;
+  title: string;
+}
+
+const HeaderWithIcon: React.FC<Props> = ({ title, description, mipNumber, link, name }) => {
+  const marginTopPopoverPosition = false;
+  const refElementShowPopover = useRef<HTMLDivElement>(null);
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const handleShowPopoverWhenNotSpace = () => {};
```

Comment:
This function do anything, could this be removed?

Reply:
this will be used for the future implementation, i will let a comment there